lib/src/gotrue_api.dart

```diff
@@ -132,7 +132,7 @@ class GoTrueApi {
     try {
       final headers = {...this.headers};
       headers['Authorization'] = 'Bearer $jwt';
-      final options = FetchOptions(headers, noResolveJson: true);
+      final options = FetchOptions(headers, noResolveJson: false);
```

Comment:
Why do you change `noResolveJson` to false ?

Reply:
Sorry, that was a mistake, I was trying to figure out the test failures / nullability of that field. I had intended on reverting it after fixing the tests, but I guess I forgot to revert before pushing.